raiden-ts/src/utils/ethers.ts

```diff
@@ -46,12 +62,23 @@ export function getLogsByChunk$(
     let retry = 3;
     // every time repeatWhen re-subscribes to this defer, yield (current/retried/next) range/chunk
     return defer(async () =>
-      provider.getLogs({
-        ...filter,
-        fromBlock: start,
-        toBlock: Math.min(start + curChunk - 1, toBlock),
-      }),
+      provider.send('eth_getLogs', [
```

Comment:
It would be good to create a feature request for ethers to support this, so e don't have to call and handle this in our code.

Reply:
I've talked to them in https://github.com/ethers-io/ethers.js/issues/1180 , but it seems they aren't going to upstream this feature, since the code which handles this is on `BaseProvider`, shared by all providers, but some children doesn't support this kind of request, e.g. `EtherscanProvider`. Since we instead require/use `JsonRpcProvider` and children, which does support it, I coded the functionality directly on this utility function which already wraps these `getLogs` calls anyway.

Reply:
Cool, thanks a lot!